### PR TITLE
Added Plaid Sandbox Link 

### PR DESCRIPTION
**Pull request by carsonwolber**
Screenshot 2024-06-30 at 11 52 37 AM

### CODE REVIEW

File: backend/flask/app.py [hunk 1]
```diff
@@ -1,54 +1,90 @@
...
+from plaid.configuration import Configuration
+import plaid
+
+# Load environment variables
+load_dotenv()
 
 app = Flask(__name__)
 CORS(app)
-app.config["SQLALCHEMY_DATABASE_URI"] = "postgresql://postgres:blu3Song85@localhost:5432/econmetrics"
+app.config["SQLALCHEMY_DATABASE_URI"] = os.getenv('DATABASE_URL')
```
Comment (pangers) [inline backend/flask/app.py:29]: Good idea moving this into env variable

**Review by pangers — APPROVED** [review]

File: frontend/src/ProfilePage/PlaidLink.jsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import React, { useState, useEffect } from 'react';
+import { usePlaidLink } from "react-plaid-link";
+import { fetchLinkToken } from '../../HelperFuncs/plaidHelp';
+
+function PlaidLink( {userId }) {
+  const [token, setToken] = useState(null);
+
+  // function to revtrieve
```
Comment (William-La) [inline frontend/src/ProfilePage/PlaidLink.jsx:8]: typo

File: frontend/src/ProfilePage/PlaidLink.jsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    }
+  }, [token]); 
+
+  const config = {
+    token,
+    env: 'sandbox',
+    onSuccess,
+  };
+
+  const { open  } = usePlaidLink(config);
```
Comment (William-La) [inline frontend/src/ProfilePage/PlaidLink.jsx:53]: nit extra space here after open

File: frontend/src/ProfilePage/PlaidLink.jsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+      if (!response.ok) {
+        throw new Error('Network response was not ok');
+      }
+      const data = await response.json();
+    } catch (error) {
+      console.error('Error exchanging public token:', error);
+    }
+  };
+
+  useEffect(() => {
```
Comment (William-La) [inline frontend/src/ProfilePage/PlaidLink.jsx:37]: clean use of useEffect!

**Review by William-La — APPROVED** [review]

File: frontend/src/ProfilePage/PlaidLink.jsx (hunk 3 above)
Comment (carsonwolber) [reply frontend/src/ProfilePage/PlaidLink.jsx:53]: fixed!

**Pull request merged**